Gemfile.lock

```diff
@@ -330,5 +330,8 @@ DEPENDENCIES
   uglifier
   warden
 
+RUBY VERSION
```

Comment:
Seems iffy. This shouldn't be locked to 2.3.0 much less to a p0 one. Maybe move it up to 2.3.1?